apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -89,6 +89,59 @@ const INITIAL_STATE = {
   warnings: [],
 }
 
+const getBlockProps = (state, currentBlock) => {
+  const {
+    amount,
+    amountToken,
+    rewardType,
+    dateReference,
+    dateStart,
+    dateEnd,
+    disbursement,
+    disbursementUnit,
+    disbursements,
+  } = state
```

Comment:
Rather than passing these all in as a single argument, can we pass them in as named arguments, aka a destructured object?

```js
const getBlockProps = ({
  currentBlock,
  amount,
  amountToken,
  rewardType,
  dateReference,
  dateStart,
  dateEnd,
  disbursement,
  disbursementUnit,
  disbursements,
}) => {
  ...
}
```